package.json

```diff
@@ -80,6 +80,8 @@
     "clean:dist": "rimraf dist/*",
     "clean:docs": "rimraf build/docs/*",
     "clean:icons": "rimraf build/icons/* src/components/Icon/icons/*.tsx",
+    "clean:illustrations": "rimraf build/illustrations/* src/components/Illustration/icons/*.tsx",
```

Comment:
Is this path correct? `src/components/Illustration/icons/*.tsx`

Reply:
Good catch, I'll update.